custom/enikshay/case_utils.py

```diff
@@ -121,3 +130,17 @@ def get_adherence_cases_between_dates(domain, person_case_id, start_date, end_da
     ]
 
     return open_pertinent_adherence_cases
+
+
+def update_case(domain, case_id, updated_properties, external_id=None):
+    kwargs = {
+        'case_id': case_id,
+        'update': updated_properties,
+    }
+    if external_id is not None:
```

Comment:
This is a shared method with the 99dots api stuff, and `external_id` isn't used there. I think we could probably allow the `external_id=None` to bubble down the call list further, but I thought this was more explicit to say that if it isn't being set in the parameters, then we shouldn't set it at all.